submissions/onyinyeChukwuka/css/sign-up.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign-up</title>
+    <link href="sign-up.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
```

Comment:
I think you should use one link to fetch your font family. Using multiple links means it will take more time for your page to load and it has to fetch from the browser, 3 times.

Reply:
@Onyinye101 Fix this